src/scripts/wp-notify.js

```diff
@@ -0,0 +1,245 @@
+    </NotifyContext.Provider>
+  );
+};
+
+class Notifications extends wp.element.Component {
+  static context = NotifyContext;
+
+  render() {
+    return (
+      <NotificationsWrap elementId="wp-notify-dashboard-notices">
+        <NotifyContext.Consumer>
+          {({ notification: notification, removeNotify }) =>
+            notification.map((notify, id) => (
+              <DashNotice
+                key={id}
+                id={id}
+                image={notify.image}
+                title={notify.title}
+                message={notify.message}
+                acceptMessage={notify.acceptMessage}
+                acceptLink={notify.acceptLink}
+                dismissLabel={notify.dismissLabel}
+                source={notify.source}
+                date={notify.date}
+                dismissible={notify.dismissible}
+                onDismiss={() => delay(100).then(() => removeNotify(id))}
+              />
+            ))
+          }
+        </NotifyContext.Consumer>
+      </NotificationsWrap>
+    );
+  }
+}
+
+// DashNotice class method.
+class DashNotice extends wp.element.Component {
+  static defaultProps = {
+    id: false,
+    image: false,
+    title: "",
+    message: "",
+    acceptMessage: __("Accept"),
+    acceptLink: "#",
+    dismissLabel: __("dismiss"),
+    source: "WordPress",
+    date: __("Just now"),
+    dismissible: false,
+    onDismiss: () => delay(100).then(() => this.unmount()),
+  };
+
+  render() {
+    let classes = "wp-notification wp-notice-" + this.props.id;
+    classes += this.props.dismissible ? " is-dismissible" : "";
+    return (
+      <div className={classes}>
+        <div className="wp-notification-wrap">
+          <h3 className="wp-notification-title">{this.props.title}</h3>
+          <p dangerouslySetInnerHTML={{ __html: this.props.message }} />
+          <div className="wp-notification-actions-wrap">
+            <a
+              className="button button-primary wp-notification-hub-trigger"
+              href={this.props.acceptLink}
+            >
+              {this.props.acceptMessage}
+            </a>
+            {this.props.dismissible && (
+              <button
+                className="button button-link wp-notification-hub-dismiss"
+                onClick={this.props.onDismiss}
+              >
+                <span className="dashicons dashicons-no-alt"></span>
+                {this.props.dismissLabel}
+              </button>
+            )}
+          </div>
+          <p className="wp-notification-source">
+            <span className="name">{this.props.source}</span> {"\u2022 "}
+            <span className="date">{this.props.date}</span>
+          </p>
+        </div>
+        {this.props.image && (
+          <div className="wp-notification-image">
+            <img src={this.props.image} alt={this.props.title + " image"} />
+          </div>
+        )}
+      </div>
+    );
+  }
+}
+
+/**
+ * THIS IS FOR TESTING PURPOSE - TO BE REMOVED
```

Comment:
```suggestion
 * TODO: THIS IS FOR TESTING PURPOSE - TO BE REMOVED
```

Not massively important, but if you specifically add the `TODO:` comment, a lot of IDEs will highlight it, makes it easier to track in my opinion.

Reply:
I did some changes to make sure that what's after that comment is purely a demo content, but I don't think we need to add a Todo, at the end it's all "to be removed" 😉